src/pykoala/ancillary.py

```diff
@@ -23,6 +25,26 @@
 # PyKOALA modules
 # =============================================================================
 
+logger = logging.getLogger('pykoala.logger')
+
+if not (logger.hasHandlers()):
+    stdout = logging.StreamHandler()
+    fmt = logging.Formatter(
+    "[PyKOALA] %(asctime)s | %(levelname)s > %(message)s")   
+    stdout.setFormatter(fmt)
+    logger.addHandler(stdout)
+    logger.setLevel(logging.INFO)
+
+def log_into_file(filename, level='INFO'):
+    logger = logging.getLogger('pykoala.logger')
+    hdlr = logging.FileHandler(filename)
+    fmt = logging.Formatter(
+    "[PyKOALA] %(asctime)s | %(levelname)s > %(message)s")   
+    hdlr.setFormatter(fmt)
+    logger.addHandler(hdlr) 
+    logger.setLevel(level)
+
+
```

Comment:
I came up with this global solution. Every module should now import the logger from this module. If the user wants to dump the log into a file, they could use the method below. Controlling the verbosity level could simply done by setting the level of the logger at any point of the pipeline.

Reply:
I'm not sure. Actually, I'm not sure how could we make sure that the PyKOALA logger
```python
logger = logging.getLogger('pykoala')
```
is initialised just once. Perhaps by setting and configuring it in <https://github.com/pykoala/pykoala/blob/main/src/pykoala/__init__.py>?

Then, children such as
```python
logger = logging.getLogger('pykoala.corrections.AtmosphericExtinction')
```
would inherit that configuration.

Moreover, one could modify it at any time, e.g.
```python
pykoala.logger.setLevel(logging.DEBUG)
```
or even for individual instances:
```python
work_in_progress.logger.setLevel(logging.DEBUG)
```

Reply:
I liked the idea of moving the parent logger to the `__init__` file, although I think it is not initialised every time you try to import it using the getter.

I have implemented the child logger for the corrections at the base level. Now, all corrections will by default create their own child, unless the user provides another logger.